### PR TITLE
Problem:(cro-1416) no address generation, unbond, withdraw, deposit in sample w…

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: refresh after sync is done

after sync is done, balance and transaction is not refreshed.
added refresh after decrypt

### CODE REVIEW

File: src/app/services/wallet.service.ts [hunk 1]
```diff
@@ -41,60 +42,76 @@ export class WalletService {
...
-    let selectedWalletId: string;
-    this.getSelectedWallet().subscribe(
-      (selectedWallet) => (selectedWalletId = selectedWallet.id)
-    );
+  async decrypt(passphrase: string, walletEnckey: string) {
+    var wallet_found: Wallet = await new Promise((resolve) => {
+      this.getSelectedWallet().subscribe((wallet) => {
+        resolve(wallet);
+      });
+    });
```
Comment (calvinaco) [inline src/app/services/wallet.service.ts:50]: Maybe use `toPromise`?
```suggestion
const wallet_found = await this.getSelectedWallet().toPromise();
```
p.s. code may not compile

File: src/app/services/wallet.service.ts [hunk 2]
```diff
@@ -41,60 +42,76 @@ export class WalletService {
...
+    await this.refresh(passphrase, walletEnckey);
+    return true;
+  }
+
+  async refresh(passphrase, walletEnckey) {
+    var wallet_found: Wallet = await new Promise((resolve) => {
+      this.getSelectedWallet().subscribe((wallet) => {
+        resolve(wallet);
+      });
+    });
```
Comment (calvinaco) [inline src/app/services/wallet.service.ts:75]: Similarly here

File: src/app/services/wallet.service.ts (hunk 1 above)
Comment (leejw51crypto) [reply src/app/services/wallet.service.ts:50]: ok, i will try

File: src/app/services/wallet.service.ts (hunk 2 above)
Comment (leejw51crypto) [reply src/app/services/wallet.service.ts:75]: ok

Comment (leejw51crypto) [reply src/app/services/wallet.service.ts]: direct await for BehaviourSubject doesn't work correctly as expected.
so added converting function

**Comment by leejw51crypto** [comment]
direct await for BehaviourSubject doesn't work correctly as expected.
so added converting function

**Comment by calvinaco** [comment]
> This pull request **introduces 1 alert** when merging [7382aab](https://github.com/crypto-com/sample-chain-wallet/commit/7382aab6440f39f2a3738d281f3d91908b55b33c) into [9bad23e](https://github.com/crypto-com/sample-chain-wallet/commit/9bad23efdbe8279974ef199621f1d90d1c8a24fc) - [view on LGTM.com](https://lgtm.com/projects/g/crypto-com/sample-chain-wallet/rev/pr-c7bbf04365a441ec7c331a40c51e19f55284923b)
> 
> **new alerts:**
> 
> * 1 for Unused variable, import, function or class

a new alert of unused variable

**Comment by leejw51crypto** [comment]
ok, i'll check

**Comment by leejw51crypto** [comment]
modifying further warnings now

**Comment by leejw51crypto** [comment]
fixed the warnings

**Pull request merged**